flink-connectors/flink-hadoop-compatibility/src/test/scala/org/apache/flink/api/hadoopcompatibility/scala/WordCountMapredITCase.scala

```diff
@@ -1,89 +0,0 @@
-/*
```

Comment:
The `WordCountMapredITCase` has a corresponding Java version, so delete it directly.